web/templates/component/button_error.html.eex

```diff
@@ -0,0 +1 @@
+<%= submit @value, class: "mr3 bn f6 link dim br-pill ph3 pv2 mb2 dib white bg-grey pointer #{@class}", id: @id, type: "button" %>
```

Comment:
Just as a note, with tachyons we can't overwrite classes by adding them on, because the rules are all defined in the css. For example, if we give a button a class of 'blue' and a class of 'red', whichever one is defined first in the css file will always be the effective class. So passing the class in to a component like this won't always be particularly useful if we're trying to overwrite the default styles we've given it